app/common/renderer/actions/Session.js

```diff
@@ -713,11 +704,15 @@ export function getSavedSessions() {
 }
 
 /**
- * Switch to a different tab
+ * Switch to a different Session Builder tab
  */
 export function switchTabs(key) {
-  return (dispatch) => {
+  return (dispatch, getState) => {
     dispatch({type: SWITCHED_TABS, key});
+    // if switching to Attach to Session tab, also retrieve the running sessions
+    if (key === 'attach') {
```

Comment:
I don't recall what `key` values could have, but could we define possible values as consts?